spec/features/collection_drafts/forms/hidden_fields_spec.rb

```diff
@@ -0,0 +1,18 @@
+# MMT-1571
+
+describe 'Removed field', js: true do
+  before do
+    login
+    draft = create(:full_collection_draft, user: User.where(urs_uid: 'testuser').first)
+    visit edit_collection_draft_path(draft)
+
+    within '.nav-top' do
+      click_on "Save"
+    end
+  end
+
+  it 'collection draft should still contain DirectoryNames' do
+    draft = CollectionDraft.first
+    expect(draft.draft.keys).to include('DirectoryNames')
```

Comment:
This test is only checking that the key still exists. I think verifying the value is unchanged would be better.